src/pybind/mgr/dashboard/frontend/src/app/shared/enum/multisite-wizard-steps.enum.ts

```diff
@@ -0,0 +1,19 @@
+export enum StepTitles {
+  CreateRealmAndZonegroup = 'Create Realm & Zonegroup',
+  CreateZone = 'Create Zone',
+  SelectCluster = 'Select Cluster',
+  Review = 'Review'
+}
+
+export const STEP_TITLES_MULTI_CLUSTER_CONFIGURED = [
+  StepTitles.CreateRealmAndZonegroup,
+  StepTitles.CreateZone,
+  StepTitles.SelectCluster,
+  StepTitles.Review
+];
+
+export const STEP_TITLES_SINGLE_CLUSTER = [
+  StepTitles.CreateRealmAndZonegroup,
+  StepTitles.CreateZone,
+  StepTitles.Review
+];
```

Comment:
why do we need enums and const for the step titles? this won't be reusable anywhere so i don't see the need for this to exist in a shared module. Maybe more fit in the rgw itself if you really think the enums and const are needed.

Reply:
Moved to multisite-wizard component